### PR TITLE
Undo toast type #14945

### PR DESCRIPTION
**Pull request by a20jonda**
Added an toast to sectioned.js where a section gets deleted.
image
Regular
image
Hover

Toast now takes 4 arguments, the last argument is a default argument and does not need to be included in every toast() call
image
The argument is used to add an method call from sectioned.js to sectioned.php.
image

### CODE REVIEW

**Review by a18chrkr — COMMENTED** [review]
Undo toast work great, both in light and dark mode. So does all the other toast. Have also tested on mobile screens. The code also looks clean.

File: Shared/css/style.css [hunk 1]
```diff
@@ -9576,6 +9576,24 @@ margin:2px;
...
+.toast .clickable .typeIcon{
+  cursor: pointer;
+  margin: 1rem 0 1rem 0;
+}
+
+.toast .clickable:hover{
+  background-color: #80eb80;
+  color: var(--color-text-label);
+  cursor: pointer;
+}
```
Comment (a22moamy) [inline Shared/css/style.css:9595]: Please add a space before every "{"-symbol to follow the K&R code standard.

**Review by a22moamy — CHANGES_REQUESTED** [review]
Look at the comment I gave, otherwise everything looks good! Will be approved once small indentation stuff is fixed.

**Review by a22moamy — APPROVED** [review]
Looks good!

**Pull request merged**